src/rngs/os.rs

```diff
@@ -1102,6 +1102,100 @@ mod imp {
     }
 }
 
+#[cfg(all(target_arch = "wasm32",
+          not(target_os = "emscripten"),
+          not(feature = "stdweb"),
+          feature = "wasm-bindgen"))]
+mod imp {
+    use core::fmt;
+
+    use __wbg_shims::*;
+
+    use {Error, ErrorKind};
+    use super::OsRngImpl;
+
+    #[derive(Clone)]
+    pub enum OsRng {
+        Node(NodeCrypto),
+        Browser(BrowserCrypto),
+    }
+
+    impl OsRngImpl for OsRng {
+        fn new() -> Result<OsRng, Error> {
+            // First up we need to detect if we're running in node.js or a
+            // browser. Both environments have `this` defined as an object for
+            // our own scope, and we use that to look up other values.
+            //
+            // Here we test if `this.self` is defined. If so we're in a browser
+            // (either main window or web worker) and if not we're in node. If
+            // it turns out we're in node.js then we require the `crypto`
+            // package and use that. The API we're using was added in Node 6.x
+            // so it should be safe to assume tha it works.
```

Comment:
Typo here, tha -> that.

Also to be pedantic the API was added in 7.x, the Node docs actually don't mention it being supported in 6.x.

They did backport it to 6.x though: https://nodejs.org/en/blog/release/v6.13.0/